llvm/test/tools/llvm-cas-object-format/csv-stats.test

```diff
@@ -0,0 +1,67 @@
+RUN: rm -rf %t && mkdir -p %t
+RUN: llc %S/Inputs/csvtest.ll --filetype=obj -o %t/csvtest.o
+
+RUN: llvm-cas-object-format --cas %t/cas --object-stats %t/stats.csv --object-stats-format=csv --ingest-schema=nestedv1 %t/csvtest.o
+RUN: cat %t/stats.csv | FileCheck %s -check-prefix=CSV
+
+RUN: llvm-cas-object-format --cas %t/cas --object-stats - --object-stats-format=csv --ingest-schema=nestedv1 %t/csvtest.o | FileCheck %s -check-prefix=CSV
+
+RUN: llvm-cas-object-format --cas %t/cas --object-stats - --object-stats-format=pretty --ingest-schema=nestedv1 %t/csvtest.o | FileCheck %s -check-prefix=PRETTY
+
+RUN: llvm-cas-object-format --cas %t/cas --object-stats %t/stats.txt --object-stats-format=pretty --ingest-schema=nestedv1 %t/csvtest.o
+RUN: cat %t/stats.txt | FileCheck %s -check-prefix=PRETTY
+
+RUN: llvm-cas-object-format --cas %t/cas --object-stats %t/stats.txt --ingest-schema=nestedv1 %t/csvtest.o
```

Comment:
It might be easier to debug the test if you use different output files for each RUN line